distributed/client.py

```diff
@@ -691,6 +714,23 @@ def shutdown(self, timeout=10):
         with ignoring(AttributeError):
             self.cluster.close()
 
+    def get_executor(self, **kwargs):
+        """ Return a concurrent.futures Executor for submitting tasks
+        on this Client.
+
+        Parameters
+        ----------
+        **kwargs:
+            Any submit()- or map()- compatible arguments, such as
+            `workers` or `resources`.
```

Comment:
People would typically use ClientExecutor to pass it to code expecting a c.f.Executor, i.e. ignorant of our additional arguments.